build-logic/publication/src/main/kotlin/com/kaspersky/kaspresso/publication/AndroidLibraryPublishExtension.kt

```diff
@@ -1,25 +0,0 @@
-package com.kaspersky.kaspresso.publication
-
-import org.gradle.api.model.ObjectFactory
-import org.gradle.api.provider.Property
-import org.gradle.kotlin.dsl.property
-import javax.inject.Inject
-
-@Suppress("UnnecessaryAbstractClass")
-abstract class AndroidLibraryPublishExtension @Inject constructor(objects: ObjectFactory) {
-
-    /**
-     * android library variant to be published
-     *
-     * default: release
-     */
-    @Suppress("UnstableApiUsage")
-    val variant: Property<String> = objects.property<String>().convention("release")
```

Comment:
I'remove this option because
* with new api impossible to simple use `afterEvaluate` to solve property usage
* we never use this feature before